api/cpp/esp-idf/slint/include/slint-esp.h

```diff
@@ -92,4 +99,5 @@ void slint_esp_init(slint::PhysicalSize size, esp_lcd_panel_handle_t panel,
  *
  * This must be called before any other call to the Slint library.
  */
-void slint_esp_init(const SlintPlatformConfiguration &config);
+void slint_esp_init(const SlintPlatformConfiguration<slint::platform::Rgb565Pixel> &config);
+void slint_esp_init(const SlintPlatformConfiguration<slint::Rgb8Pixel> &config);
```

Comment:
Should there be documentation for the new function?

Reply:
I mean, I can copy it, but it seems pointless for what's an overload. But when we generate docs in the future, we need to make sure that it shows up as one function or two with the same docs.